charts/sftpgo/values.yaml

```diff
@@ -56,6 +56,15 @@ envFrom: []
 # See the [API reference](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#environment-variables) for details.
 env: {}
 
+# -- Additional environment variables values of which are populated from specific [secrets and keys](https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-environment-variables)
+envFromSecrets: []
```

Comment:
Let's call this `envVars` instead.

Reply:
@sagikazarmark  made the needed changes please review